src/docker-it/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:14.04
+
+RUN apt-get install -y ssh && mkdir /var/run/sshd && chmod 0755 /var/run/sshd
```

Comment:
@vkuchyn you missed this one :) Good thing you did though, we also need to lock in the version on ssh anyhow.
Just instead of `apt-get install -y ssh` put `apt-get update && apt-get install -y ssh=1:6.6p1-2ubuntu2.6`